releasenotes/notes/num-qubits-streamlining-01a9f58684ca4609.yaml

```diff
@@ -0,0 +1,19 @@
+---
+deprecations:
+  - |
+    The ``n_qubits`` and ``numberofqubits`` keywords are deprecated throughout
+    Terra and replaced by ``num_qubits``. The objects affected by this change are listed below.
+
+    class                  | old method      | new method
+    -----------------------+-----------------+--------------------------------
+    QuantumCircuit         | n_qubits        | num_qubits
+    Pauli                  | numberofqubits  | num_qubits
+    BackendConfiguration   | nothing removed | num_qubits (additionally added)
```

Comment:
This would be a new feature not a deprecation if we wanted to advertise. I'm not sure it's really a change for something that we need to document to users as a change in the next release.

Reply:
Updated 👍

Reply:
You actually were closer before, the other lines should be a deprecation. I specifically meant that the `BackendConfiguration` line in this table is not a deprecation. You're adding a new attribute/property to the class. The others are definitely deprecations

Reply:
Ah, now I understood. Yes that makes more sense 🙂